src/modules/ModuleLiquidity/store/add.ts

```diff
@@ -170,7 +171,24 @@ export const useLiquidityAddStore = defineStore('liquidity-add', () => {
 
   // #region Selection
 
-  const selection = usePairInput({ addrsOrigin: useLocalStorageAddrsOrigin('liquidity-add-selection', isActiveRoute) })
+  const routeAddrsOrigin = useRouteAddrsOrigin({
+    baseToken: NATIVE_TOKEN_FULL.address,
+    additionalBaseToken: DEX_TOKEN_FULL.address,
+    isActive: isActiveRoute,
+  })
+  const localStorageAddrsOrigin = useLocalStorageAddrsOrigin('liquidity-add-selection', isActiveRoute)
+
+  const addrsOrigin = computed({
```

Comment:
Maybe it makes sense to sync local storage with route params?

```ts
watch(routeAddrsOrigin, (x) => {
  // if `x` is not the same as in the local storage, update local storage
}, { immediate: true }) // immediate, so at the init stage it will take priority

watch(localStorageAddrsOrigin, (x) => {
  // if `x` is not the same as route params... remove route params?
})
```

In that case, `localStorageAddrsOrigin` could be considered as a source of truth for `usePairInput()`.

Reply:
Yeah, good idea!

Reply:
Done